### ISSUE TITLE
First-run onboarding experience - signup and app install

### ISSUE THREAD

**Issue by kleneway** [jacob-ai-bot/jacob#56]
Based on initial user feedback, we need to improve the initial onboarding experience for new users. There are a few main goals for this work

- Guide users through the minimum required steps
  - Sign up via GitHub and provide an email address
  - Install the JACoB GitHub app
  - Connect their first repository (either a new sample repo or an existing one)
- Help users understand the basics of using JACoB in a "show, don't tell" type of experience
- Give users an early "wow" moment 

This can be accomplished by creating a new onboarding Developer persona that uses the chat interface to dynamically present the required signup / connection buttons, and then use the scenario prompt to gather the info necessary to complete the jacob.config file. The conversation will end by creating a new issue that instructs JACoB to create a new `jacob.config` file in the repo and file a PR when it is complete. Once the PR is merged, the initial onboard is complete.

- [ ] Update the Signup page to collect an email address. Use the email from GitHub as the default (if available) but allow them to change it
- [ ] Create a new Developer persona for setup
- [ ] Write the scenario prompt that will do the following:
  - [ ] Explain the basics of what JACoB does (and doesn't yet do)
  - [ ] Give the user a choice of connecting an existing repo or cloning a sample one
     - [ ] If the user chooses an existing, show a button that takes them to the GitHub app installation page
     - [ ] If the user chooses to use a sample one, clone the `t3-sample` repo and add it to their org. Then show the button that takes them to the GitHub app installation page and tell them to select the sample repo. 
   - [ ] Ask the user if they use Figma. If they do, give a button to redirect them to install the Figma plugin 
  - [ ] When the user returns, present a button that will check the JACoB installation status and report back whether or not JACoB was successfully installed
     - [ ] If it is installed, show a congrats message and a button that will take them to the `create-jacob-config` developer experience
     - [ ] If it is not installed successfully, give instructions on what went wrong. If it's due to an incompatible language, prompt them to install the sample repo instead. Otherwise give them the error message with instructions to fix it and try again.
   - [ ] Mark the user as `onboarded` in the database
